presto-docs/src/main/sphinx/connector/mysql.rst

```diff
@@ -215,16 +215,110 @@ The connector maps PrestoDB types to the corresponding MySQL types:
 
 No other types are supported.
 
+SQL Support
+-----------
+
+The MySQL connector allows querying and creating MySQL tables. Here are some examples of the SQL operations supported:
+
+ALTER TABLE
+^^^^^^^^^^^
+
+.. code-block:: sql
+
+    ALTER TABLE mysql.web.page_views ADD COLUMN zipcode VARCHAR;
+    ALTER TABLE mysql.web.page_views RENAME COLUMN zipcode TO location;
+    ALTER TABLE mysql.web.page_views DROP COLUMN location;
+
+CREATE TABLE
+^^^^^^^^^^^^
+
+Create a new MySQL table named ``page_views`` in the ``web`` schema:
+
+.. code-block:: sql
+
+    CREATE TABLE mysql.web.page_views (
+      user_id bigint,
+      page_url varchar,
+      ds date,
+      country varchar
+    );
+
+.. note:: Presto does not enforce primary key constraints.
```

Comment:
```suggestion
.. note:: Presto does not enforce primary key constraints. For example, the following statement 

  .. code-block:: sql

   CREATE TABLE users (
           id INT PRIMARY KEY,
           name TEXT,
           email TEXT
       );

  returns an error similar to the following: 

  ``Query 20240322_095447_00010_syzb3 failed: line 2:19: mismatched input 'PRIMARY'. Expecting: ')', ','``
```

When I asked why you removed the error blocks, I didn't intend you to put them back in as is, I only wanted to understand why you had removed them. I think the new error blocks here are an improvement from your previous draft, but they could be improved and I want to explain why and how.

One problem with the errors you include is that the example error code (in lines 248-254) isn't explained to the user as a statement that outputs the error in the block following. The user must read the Note, then read the code block and figure out that this code block contains a primary key, then that this error - which is loud and the user's eye goes to it first - only happens with primary keys in the context of this kind of statement. 

How to fix? Let's include the error code block, and the error (not the error block, just the error) into the Note. The user can read the first sentence in the Note about primary keys and if that doesn't matter to them they can stop reading. 

I suggest you replace lines 246-259 with the suggestion I've written out here, which I tested in my local build and if I wrote it already you don't need to reverse-engineer the RST code from the screenshot of my local build, which I will also include: 

<img width="1071" alt="Screenshot 2024-03-22 at 12 10 13 PM" src="https://github.com/prestodb/presto/assets/7013443/80f62442-c745-4908-995f-307d450ffb86">

Notice the screenshot includes the Note generated from the suggested code, and also includes the good code example above it and the CREATE TABLE AS SELECT heading below it to show how the Note begins and ends.